featurebyte/query_graph/sql/adapter/bigquery.py

```diff
@@ -0,0 +1,27 @@
+"""
+BigQueryAdapter class
+"""
+
+from __future__ import annotations
+
+from sqlglot.expressions import Anonymous, Expression
+
+from featurebyte.enum import SourceType
+from featurebyte.query_graph.sql.adapter.snowflake import SnowflakeAdapter
+
+
+class BigQueryAdapter(SnowflakeAdapter):
```

Comment:
Is it intended to inherit from `SnowflakeAdapter`? The datatypes are a little different for BigQuery (e.g. no timestamp with timezones)

Reply:
Yes, this is intended temporarily. This allows us to instantiate the adapter before the remaining methods are implemented.